### ISSUE TITLE
Question on the RL performance on wan2.2.

### ISSUE THREAD

**Issue by chenkang455** [Tencent-Hunyuan/UniRL#16]
Hi Hunyuan Team,

Thanks for open-sourcing this great work! I was wondering where I could find the evaluation results for the RL-finetuned models, such as Wan2.2. Is there a table comparing the performance before and after RL fine-tuning?

Thanks again!

**Comment by Jayce-Ping** [comment]
Since video models are heavy and the rollout is also expensive, it takes a while to see the fully converage with the latest codebase. We will update the results later in this channel.

**Comment by Jayce-Ping** [comment]
Based on the latest codebase, running `diffusion/wan22_t2v_14b` provides the following training curve:

<img width="467" height="301" alt="Image" src="https://github.com/user-attachments/assets/06831831-dfd4-42f7-9cc2-028d33707d28" />

where the reward is simply computed by PickScore. You can try different setting with different reward. Since training for video models is too expensive, the progress of more various settings is slow. We will add more later on. If you have any interests in it, feel free to open PR for verified reward or config.

**Comment by chenkang455** [comment]
@Jayce-Ping Got it, thanks for sharing. 🎉🎉🎉
